contracts/modules/VersionManager.sol

```diff
@@ -113,6 +115,15 @@ contract VersionManager is IVersionManager, IModule, BaseFeature, Owned {
         walletFactory = _factory;
     }
 
+    /**
+     * @notice Lets the owner change the minimum allowed version
+     * @param _minVersion the minimum allowed version
+     */
+    function setMinVersion(uint256 _minVersion) external onlyOwner {
+        require(_minVersion > 0 && _minVersion <= lastVersion, "VM: invalid _minVersion");
```

Comment:
Shouldn't it be `_minVersion > minVersion` to prevent us from potentially reverting to an incorrect state?

Reply:
Potentially but I thought of leaving the owner the flexibility of "re-whitelisting" a previously blacklisted version in case this was ever needed.